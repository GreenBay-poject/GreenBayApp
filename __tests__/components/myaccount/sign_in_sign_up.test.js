import React from 'react';
import renderer from 'react-test-renderer';
import SignInForm from '../../../components/myaccount/sign_in_sign_up';

describe('<SignInForm />', () => {
    it('Sample SignInForm', () => {
        const tree = renderer.create(<SignInForm />).toJSON();
        console.log(tree)
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('RCTSafeAreaView');
        expect(tree.children[0].type).toBe("Text");
        expect(tree.children[1].type).toBe("TextInput");
        expect(tree.children[2].type).toBe("TextInput");
        expect(tree.children[3].type).toBe("View");
        expect(tree.children[4].type).toBe("Text");


    });
});