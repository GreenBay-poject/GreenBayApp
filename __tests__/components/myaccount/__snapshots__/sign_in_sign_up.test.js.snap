// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInForm /> Sample SignInForm 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#d7dbd8",
        "color": "#153d37",
        "fontSize": 26,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    SIGN IN
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Email"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "stretch",
        "borderWidth": 1,
        "height": 40,
        "margin": 12,
        "padding": 10,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Password"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "alignSelf": "stretch",
        "borderWidth": 1,
        "height": 40,
        "margin": 12,
        "padding": 10,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#153d37",
        "elevation": 4,
        "marginBottom": 10,
        "marginHorizontal": 20,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          disabled={false}
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#153d37",
              },
            ]
          }
        >
          SIGN IN
        </Text>
      </View>
    </View>
  </View>
  <Text
    onPress={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "color": "#0e0463",
        "padding": 10,
      }
    }
  >
    Don't Have Account
  </Text>
</RCTSafeAreaView>
`;
